ChopChop/Views/Recipes/RecipeCollectionView.swift

```diff
@@ -152,24 +152,18 @@ struct RecipeCollectionView: View {
     func GridTileOverlay(recipe: RecipeInfo) -> some View {
         ZStack(alignment: .bottomLeading) {
             Rectangle()
-                .fill(
-                    LinearGradient(
-                        gradient: Gradient(stops: [
-                            .init(color: Color.black.opacity(0), location: 0.3),
-                            .init(color: .black, location: 0.6)
-                        ]),
-                        startPoint: .top,
-                        endPoint: .bottom
-                    )
-                )
+                .foregroundColor(.clear)
+                .background(LinearGradient(gradient: Gradient(colors: [.clear, .black]),
+                                           startPoint: .top,
+                                           endPoint: .bottom))
```

Comment:
Is using a background with a clear foreground better than using fill?

Reply:
Actually the difference in light/dark mode might affect how the overlay looks, but regardless I don't think it's a big issue as long as the user can see the light text against the dark background.